client/rest-high-level/src/test/java/org/elasticsearch/client/core/TermVectorsResponseTests.java

```diff
@@ -158,14 +158,14 @@ protected TermVectorsResponse createTestInstance() {
             terms.add(randomTerm(hasTermStatistics, hasScores, hasOffsets, hasPositions, hasPayloads));
         }
 
-        TermVectorsResponse.TermVector tv = new TermVectorsResponse.TermVector("field" + randomAlphaOfLength(2), fs, terms);
+        TermVectorsResponse.TermVector tv = new TermVectorsResponse.TermVector("field" + randomAlphaOfLength(7), fs, terms);
```

Comment:
I think it'd be nice to check for duplicated before adding to the list and retry. Or to pass in a unique prefix to this method to be sure we don't have duplicates.